test/PR_test/integration_test/op/tensorop/test_un_hadamard.py

```diff
@@ -42,9 +42,11 @@ def test_torch_4class(self):
         output = fromhadamard.forward(
             data=[
                 torch.tensor([[1., -1., -1., 1.], [-1., 1., -1., -1.], [-1., 1., -1., -1.], [-1., 1., -1., -1.],
-                              [-1., 1., 1., 1.]])
+                              [-1., 1., 1., 1.]]).to("cuda:0" if torch.cuda.is_available() else "cpu")
```

Comment:
why are we removing the gpu test?

Reply:
I don't see where we are removing gpu test

Reply:
where are we removing gpu test?

Reply:
oh lol. read it backwards